src/meson.build

```diff
@@ -1,9 +1,28 @@
 sources = [
-    'models/Package.vala',
     'main.vala',
 ]
 
-executable('vamp', sources,
+ vamp_lib = library(
+    'vamp',
+    [
+        'models/Bugs.vala',
+        'models/FundingInfo.vala',
+        'models/Package.vala',
+        'models/Person.vala',
+        'models/Repository.vala',
+    ],
+    vala_header: 'vamp.h',
+    vala_vapi: 'vamp.vapi',
     dependencies: dependencies,
+)
```

Comment:
If we're going to have a library for this, I think it would be better to have separated folders: `vamp` and `cli` :thinking: